src/test/java/com/opentable/components/filterorder/CycleFinderTest.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.opentable.components.filterorder;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+
+import org.assertj.core.api.Assertions;
+import org.junit.Test;
+
+public class CycleFinderTest {
+    @Test
+    public void testTrivial() {
+        final Optional<List<Object>> cycle = new CycleFinder<>(Collections.emptyMap()).run();
+        Assertions.assertThat(cycle).isNotPresent();
```

Comment:
Why not do a static import?

Reply:
To avoid filling up the file's namespace.